app/assets/javascripts/Components/assignment_summary_table.jsx

```diff
@@ -98,6 +98,37 @@ class AssignmentSummaryTable extends React.Component {
       filterable: false,
       defaultSortDesc: true,
     },
+    {
+      Header: I18n.t('activerecord.models.tag.other'),
+      accessor: 'tags',
+      Cell: row => (
+        <ul className="tag-list">
+          {row.original.tags.map(tag =>
+            <li key={`${row.original._id}-${tag}`}
+              className="tag-element">
+              {tag}
+            </li>
+          )}
+        </ul>
+      ),
+      minWidth: 80,
+      sortable: false,
+      filterMethod: (filter, row) => {
+        if (filter.value) {
+
+          // Check tag names
+          const tag_matches = row._original.tags.some(
+            (tag) => tag.includes(filter.value)
+          );
+
+          if (tag_matches) {
+            return true;
+          }
```

Comment:
This is quite nice, but can be simplified to just `return row._original.tags.some(...);`.